### PR TITLE
Fixed scap cli tests and added coverage for scap-ansible

### PR DESCRIPTION
**Pull request by san7ket**
```
 pytest tests/foreman/cli/test_oscap.py
============================================================================================================ test session starts =============================================================================================================
platform linux -- Python 3.6.8, pytest-4.0.2, py-1.6.0, pluggy-0.9.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/sjagtap/PycharmProjects/robottelo/tests/foreman, inifile: pytest.ini
plugins: xdist-1.25.0, services-1.3.1, mock-1.10.0, forked-0.2, cov-2.6.0
collecting ... 2019-05-15 20:31:36 - conftest - DEBUG - Collected 35 test cases

collected 35 items                                                                                                                                                                                                                           

tests/foreman/cli/test_oscap.py ................s....s..s..........                                                                                                                                                                    [100%]


============================================================================================= 32 passed, 3 skipped, 8 warnings in 952.47 seconds =======
```

### CODE REVIEW

**Comment by san7ket** [comment]
Partially fixes #6930

**Comment by san7ket** [comment]
@ares @oprazak could give give it a look , these are just partial tests, Will come up with E2E soon

**Comment by xprazak2** [comment]
LGTM

File: tests/foreman/cli/test_oscap.py [hunk 1]
```diff
@@ -673,10 +680,48 @@ def test_positive_associate_scap_policy_with_hostgroups(self):
...
+            2. Oscap-cli hammer plugin installed.
+            3. More than 1 hostgroups
+            4. Ansible role and Ansible variable
+
+        :steps:
+
+            1. Login to hammer shell.
+            2. Execute "policy" command with "create" as sub-command.
+            3. Pass valid parameters and deploy option as ansible
+            4. Associate multiple hostgroups with policy
```
Comment (omkarkhatavkar) [inline tests/foreman/cli/test_oscap.py]: I'm not sure if I understand correctly, are you apply multiple host groups here?  I see only one host group, not a list. please correct me if I'm wrong

**Pull request merged**